docs/support/tutorials/data-mover.md

```diff
@@ -0,0 +1,72 @@
+# Data-mover
+
+Data-mover is a tool to move data between Puhti and Mahti local filesystems and
+Allas and LUMI-O object storage servers, when
+[simple transfers](../faq/how-to-move-data-between-puhti-and-allas.md#move-data-with-rclone)
+are not practical, either because there are many small files, or the size of the
+dataset is large.
+
+We wish the data-mover tool `dm` to be simple to use, and handle all possible
+hard corner cases. It is basically a wrapper around [Restic backup tool](https://restic.readthedocs.io)
+, and stores the data in Restic repositories.
+Restic in turn uses [Rclone](https://rclone.org) for the actual data transfers to
+the object storage servers and back. In addition, the data-mover tool does the
+data transfers in the background, using batch jobs, allowing larger transfers
+than would be practical in regular interactive login sessions.
+
+Below is a guide for a simple scenario, moving data from Puhti project scratch
+directory to corresponding project in Allas, and then back. Similar works with
```

Comment:
similar what works? Unclear

Reply:
I guess you mean exact same instructions work on mahti, which is true. Lumi-O is slightly different, need to specify more

Reply:
Ok. I reread the doc. Similar is exactly how it is. Very unclear, but truthfully so :)

Reply:
you could just say that easy instructions work the same way in Mahti. Cross service usage, using Lumi-O instead of allas is possible, please go read the advanced section if you are interested